components/webapps/operator-onboard/src/main/java/com/wso2telco/dep/service/OperatorService.java

```diff
@@ -36,4 +39,33 @@ public Response addOperator(String jsonBody) throws Exception{
 		
 		return null;	
 	}
+
+	public void fooMethod(String abc) {
+
+		try {
+			Thread.sleep(1000);
+			FileInputStream file = new FileInputStream("hard_coded_path");
+		} catch (InterruptedException e) {
+
+		} catch (FileNotFoundException e) {
+			e.printStackTrace();
+		}
+
+	}
+
+	public String barMethod(String abc) {
+
+		try {
+			Thread.sleep(1000);
+
+
+			FileInputStream file = new FileInputStream("hard_coded_path");
+		} catch (InterruptedException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3AS2142)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3AS00108)